### PR TITLE
Handle Enter key in Bootstrap modal dialogs

### LINKED ISSUE #863: Bootstrap dialog boxes should handle a default action with Enter key
Currently, when a bootstrap dialog box is displayed, hitting Enter key has no effect.

I think it should trigger the default action: OK when it's only an info, Confirm when it's a confirmation request.

### PR DESCRIPTION
**Pull request by mossroy**
Fixes #863

### CODE REVIEW

File: www/js/lib/uiUtil.js [hunk 1]
```diff
@@ -73,6 +73,19 @@ define(rqDef, function(settingsStore) {
                     resolve(false);
                 }
             });
+            // Enter key should accept default action
+            $("#alertModal").keydown(function(e){
```
Comment (Jaifroid) [inline www/js/lib/uiUtil.js:77]: I wonder if keyup is more usual, i.e. you expect the event to occur after you have released the key? Main reason to use keydown is if you're trying to get in first before other event listeners.

I notice you have used jQuery here. Is this to match the style of similar keydown blocks we already have?

In line with #367, I now try to use only native DOM methods. But there's an argument for matching coding style of nearby blocks.

Otherwhise:
```
document.getElementById('alertModal').addEventListener('keyup', function (e) {
    ...
}
```

**Review by Jaifroid — APPROVED** [review]
I tested, and it works. I may be wrong about keyup vs keydown.

PS Tested in Chromium (Edge) and IE11 mode.

File: www/js/lib/uiUtil.js (hunk 1 above)
Comment (mossroy) [reply www/js/lib/uiUtil.js:77]: Many thanks for this feedback.
The choice of keyup/keydown/keypress event was not really thought about, on my side. No problem to switch to keyup.

I used jQuery to match the style of the code above it, but I fully agree to switch to native DOM methods. I will change the code

Comment (Jaifroid) [reply www/js/lib/uiUtil.js:77]: > The choice of keyup/keydown/keypress event was not really thought about, on my side. No problem to switch to keyup.

Use whatever seems most natural to you. I'm not very convinced by my own reasoning last night!

**Comment by Jaifroid** [comment]
I tested the native DOM code in IE11. Working fine. I think it can be merged.

**Pull request merged**